lib/evss/vso_search/service.rb

```diff
@@ -31,6 +31,13 @@ def get_current_info(additional_headers = {})
       def request_headers(additional_headers)
         ssn = additional_headers.key?('va_eauth_pnid') ? additional_headers['va_eauth_pnid'] : @user.ssn
         edipi = additional_headers.key?('va_eauth_dodedipnid') ? additional_headers['va_eauth_dodedipnid'] : @user.edipi
+        if edipi.nil?
+          user_info = {
+            loa3: @user.loa3?,
+            mhv_icn: !@user.mhv_icn.nil?
```

Comment:
The goal is to just get boolean values for both of these, right? You can also do `@user.mhv_icn.present?`, but if it's temporary I guess it doesn't matter